Added actual QR code reading.

netaneld122 opened the pull request:
Added requirements.txt for dependencies.
Added .gitignore.
Updated README according to new usage.

Commit:
Added actual QR code reading.

Added requirements.txt for dependencies.
Added .gitignore.
Updated README according to new usage.

yuvadm commented on .gitignore:
```diff
@@ -0,0 +1,141 @@
+cython_debug/
+
+# Pycharm
+.idea/
```
This is a huge .gitignore - I'd minimize it to just `*.pyc` and `__pycache__`

netaneld122 replied:
Fixed

yuvadm commented on README.md:
````diff
@@ -8,7 +8,16 @@ As (not really) specified in https://github.com/MohGovIL/Ramzor
-It requires the [`cryptography`](https://pypi.org/project/cryptography/) package to be installed.
+Run the following command to install the dependency packages:
+```bash
+pip install -r requirements.txt
````
I recommend upgrading the entire flow to use `pipenv` which is a slightly more modern approach

netaneld122 replied:
Fixed

yuvadm commented on README.md:
````diff
@@ -8,7 +8,16 @@ As (not really) specified in https://github.com/MohGovIL/Ramzor
+### Usage
+Save your Green Pass as a `.png` file and execute:
+```bash
+python verify.py path_to_my_green_pass_image.png
````
We should probably accept both images and pre-decoded payloads

netaneld122 replied:
Fixed

yuvadm commented on requirements.txt:
```diff
@@ -0,0 +1,3 @@
+cryptography
+pillow
+pyzbar
```
Replace with `Pipfile`

netaneld122 replied:
Fixed

yuvadm commented on verify.py:
```diff
@@ -1,28 +1,55 @@
-digest = h.finalize()
+def cert(name):
+    this_dir = os.path.dirname(os.path.abspath(__file__))
+    return os.path.join(this_dir, 'certs', name)
```
A nicer approach would be to use `pathlib` for concat'ing paths

netaneld122 replied:
Fixed

yuvadm reviewed (changes requested):
Thanks for the contribution @netaneld122 ! Added some comments

Commit:
Fixed CR notes.

Migrated requirements to a Pipfile format.
Added ability to open textual payload.
Removed irrelevant paths in .gitignore.

Commit:
Fixed pipenv run cli.

Commit:
Merge branch 'main' of github.com:yuvadm/greenpass into feature/read-qr-code-from-image

# Conflicts:
#	verify.py

yuvadm reviewed (approved):
Looks good! Some small changes needed but I'll handle those after merging. Thanks @netaneld122 :pray:
